### DISCUSSION TITLE
Student ID upload keeps failing

### DISCUSSION THREAD

**Discussion by Keith-Web3** [community/community#110247] [GitHub Education]
### Hi everyone,

I'm trying to apply for the GitHub student pack. I'm currently at the stage where I need to upload proof of my academic status. The picture of my ID card that I tried to upload is about 750kb but each time I click on 'process application', I get an error in my console saying the request payload is too high

**Comment by deonlobo** [comment]
If you are facing challenges due to the size of the request payload when attempting to upload proof of your academic status for the GitHub Student Pack, consider the following suggestions:

1. **Reduce Image Size:**
   - Compress the image before uploading to reduce its file size.
   - Use an image editor or an online image compression tool to reduce the file size while maintaining readability.

2. **Supported File Formats:**
   - Ensure that you are uploading an image in a supported format (e.g., JPEG, PNG).
   - Verify if GitHub has any specific requirements regarding the format of the uploaded file.

3. **Check File Type Restrictions:**
   - Confirm that the platform accepts the file type of your image (e.g., ID card image).
   - Some platforms have restrictions on the types of files that can be uploaded.

4. **Alternative Formats:**
   - If possible, try uploading a different format (JPEG instead of PNG, or vice versa) to see if that resolves the issue.

5. **Try Different Browsers:**
   - Attempt the upload using a different web browser. Sometimes, browser-specific issues can affect file uploads.

6. **Contact GitHub Support:**
   - If the issue persists, consider reaching out to GitHub Support for assistance. They may be able to provide guidance on resolving the specific error you are encountering.

**Comment by mishmanners** [comment]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:

- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB
